README.md

```diff
@@ -1,5 +1,29 @@
-# Momento client-types-php
+# client-sdk-php
 
-:warning: Pre-Alpha :warning:
+:warning: Experimental SDK :warning:
+
+PHP SDK for Momento is experimental and under active development. There could be non-backward compatible changes or
+removal in the future. Please be aware that you may need to update your source code with the current version of the SDK
+when its version gets upgraded.
+---
+
+<br />
+PHP SDK for Momento, a serverless cache that automatically scales without any of the operational overhead required by
+traditional caching solutions.
+
+<br/>
+
+# Getting Started :running:
+
+## Requirements
+
+- A Momento Auth Token is required, you can generate one using the [Momento CLI](https://github.com/momentohq/momento-cli)
+- At least PHP 7
+- The grpc PHP extension. See the [gRPC docs](https://github.com/grpc/grpc/blob/v1.46.3/src/php/README.md) section on installing the extension.
+
+## Using Momento
+
+Run `composer install` from the root of the repository to install the necessary prerequisites.
+
+Then check out full working code in `example.php` at the root of this repository!
```

Comment:
nit, in subsequent PRs let's just move this into an examples subdir because there will be more than one over time

Reply:
I'm totally planning on that, but the way composer works I need to get this PR merged before I can hook that up cleanly. Moving the script into it's own directory with it's own `composer.json` to grab the entire SDK is my next step for this after merge.